src/test/resources/release-notes-file.md

```diff
@@ -0,0 +1,11 @@
+### New features
+
+* [JIRA-TICKET](jira-link): Some feature
```

Comment:
Wait, so RELEASE-NOTES.md will be empty after every release?

That's opposite to its original intent, which was to have the entire project history neatly in one place, along with the code (and not somewhere else entirely, like JIRA or GitHub's web site).